serving/src/main/java/feast/serving/config/ServingServiceConfigV2.java

```diff
@@ -77,6 +82,20 @@ public ServingServiceV2 servingServiceV2(
         OnlineRetrieverV2 bigtableRetriever = new BigTableOnlineRetriever(bigtableClient);
         servingService = new OnlineServingServiceV2(bigtableRetriever, specService, tracer);
         break;
+      case CASSANDRA:
+        CassandraStoreConfig config = feastProperties.getActiveStore().getCassandraConfig();
+        String host = config.getHost();
+        Integer port = config.getPort();
+        String dataCenter = config.getDataCenter();
+
+        CqlSession session =
```

Comment:
nit: Perhaps this part could be initialized within the storage connector instead.